app/models/embed/purl/resource_file.rb

```diff
@@ -73,6 +73,10 @@ def stanford_only_downloadable?
         download == 'stanford'
       end
 
+      def not_no_download?
```

Comment:
Since `no_download?` already exists, can you call that one instead (e.g. `!no_download?`) and remove this method?

Reply:
done